plugins/org.jkiss.dbeaver.ext.wmi/OSGI-INF/l10n/bundle_uk.properties

```diff
@@ -0,0 +1,11 @@
+#Generated by ResourceBundle Editor (http://essiembre.github.io/eclipse-rbe/)
+dialog.connection.header=\u041D\u0430\u043B\u0430\u0448\u0442\u0443\u0432\u0430\u043D\u043D\u044F \u043F\u0456\u0434\u043A\u043B\u044E\u0447\u0435\u043D\u043D\u044F WMI
+
+meta.org.jkiss.dbeaver.ext.wmi.model.WMIClass.superClass.name=\u0411\u0430\u0442\u044C\u043A\u0456\u0432\u0441\u044C\u043A\u0438\u0439 \u043A\u043B\u0430\u0441
```

Comment:
I think, that it can be just "Супер клас". Yes, it is a kind of parent class, but super - is simpler.

Reply:
I will change it, but we will have to see in what context it will be used. Because maybe the first option will be more understandable for us, because for us "Супер" can still mean something incredible, extraordinary, unsurpassed and so on